packages/shared/src/domain/patient.ts

```diff
@@ -2,7 +2,7 @@ import { z } from "zod";
 
 export const patientSchema = z.object({
   id: z.string(),
-  externalId: z.string().optional(),
+  externalId: z.string().optional().or(z.null()),
```

Comment:
The patient DTO seems to return `null` for certain patients where there is no external ID set, also flagging this for anyone else that is validating patients against this schema.